include/dem/particle_particle_contact_force.h

```diff
@@ -865,16 +872,17 @@ class ParticleParticleContactForce
   inline void
   calculate_hertz_contact(
     particle_particle_contact_info<dim> &contact_info,
-    const double                         normal_relative_velocity_value,
-    const Tensor<1, 3>                  &normal_unit_vector,
-    const double                         normal_overlap,
-    const ArrayView<const double>       &particle_one_properties,
-    const ArrayView<const double>       &particle_two_properties,
-    Tensor<1, 3>                        &normal_force,
-    Tensor<1, 3>                        &tangential_force,
-    Tensor<1, 3>                        &particle_one_tangential_torque,
-    Tensor<1, 3>                        &particle_two_tangential_torque,
-    Tensor<1, 3>                        &rolling_resistance_torque)
+    const Tensor<1, 3> & /*tangential_relative_velocity*/,
```

Comment:
We should add a TODO for the Hackathon. It is so weird to not have a tangential damping for the basic hertz model.